privcount/data_collector.py

```diff
@@ -656,8 +670,25 @@ def handle_event(self, event):
             if len(items) == 6:
                 self._handle_connection_event(items[0:6])
 
+        elif event_code == 'PRIVCOUNT_STREAM_BYTES_TRANSFERRED':
+            # 'PRIVCOUNT_STREAM_BYTES_TRANSFERRED', ChanID, CircID, StreamID, BW, Direction, Time
+            if len(items) == 6:
+                self._handle_bytes_event(items[0:6])
+
         return True
 
+    def _handle_bytes_event(self, items):
+        if self.traffic_model == None:
+            return
+
+        chanid, circid, strmid = [int(v) for v in items[0:3]]
+        direction = items[3]
```

Comment:
Is there any reason why direction is a string?
Other flags (and other directions) in events are 1 or 0.
See issue #189: if we decide to fix it, we should fix it before we merge this code.